### DISCUSSION TITLE
How do I pass the provisioning profile as a command line argument?

### DISCUSSION THREAD

**Discussion by louwers** [bazelbuild/rules_apple#1957] [Q&A] [answered]
Is it possible to pass the provisioning profile to use as a command line argument?

I can only find `--ios_signing_cert_name=...`

**Comment by keith** [comment, marked as answer]
It's not, what's your use case? At least one problem is if you're building multiple targets at once it wouldn't work. Maybe you want to instead use a `select()` on the target and pass a `--define` (or switch based on `--compilation_mode`)

**Reply by louwers** [reply]
My use case is that different users will probably have a differently named provisioning profile. And on top of that, if a build for the simulator is made a provisioning profile is not needed at all.

> Maybe you want to instead use a select() on the target and pass a --define (or switch based on --compilation_mode)

That is an interesting option! I wasn't aware of these Bazel features, but I will have a closer look.

I think another option is to just create some more targets at the expense of some duplication.
